app/controllers/users/mfa_selection_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Users
+  class MfaSelectionController < ApplicationController
+    include UserAuthenticator
+    include MfaSetupConcern
+
+    before_action :authenticate_user
+    before_action :confirm_user_authenticated_for_2fa_setup
+    before_action :multiple_factors_enabled?
+
+    def index
+      @two_factor_options_form = TwoFactorOptionsForm.new(current_user)
+      @presenter = two_factor_options_presenter
+      analytics.track_event(Analytics::USER_REGISTRATION_2FA_ADDITIONAL_SETUP_VISIT)
```

Comment:
the new style of analytics methods would look more like:
```suggestion
      analytics.user_registration_2fa_additional_setup_visit
```